admin/index.php

```diff
@@ -243,7 +243,9 @@
 if ( !$is_bookmark ) { ?>
 	<p><?php echo $search_sentence; ?></p>
 	<p><?php
-		printf( yourls__( 'Display <strong>%1$s</strong> to <strong class="increment">%2$s</strong> of <strong class="increment">%3$s</strong> URLs' ), $display_on_page, $max_on_page, $total_items );
+        if ($total_items === 0)
+            printf( yourls__( 'Displaying <strong>ZERO</strong> URLs, because none have been defined' ));
```

Comment:
This can also appear if the user applied a filter with no matching results. "because none have been defined" is only accurate for one specific case.

```suggestion
            printf( yourls__( 'Displaying <strong>0</strong> URLs matching the current filters' ) );
```

Perhaps there's a better way to phrase it, or if you think it's worth the effort you could add a check if any filter conditions are set and display a different message.

Reply:
Thank you for the additional information. First time user here, I have less than 10 links in my setup and not using filters yet...

So, my proposal: let's use a wording that is universal while still useful. I'm thinking: "no URLs to display." Would that work for you?

Reply:
I've come to really like the idea of indicating to the user that filters are active, but "No URLs to display" does work! Short, sweet, and simple.

Reply:
Well all right then, that decision is totally up to you. It is called a pull request not coercion. Requests can be denied! 😊

Reply:
No I'm not sure I understood your comment properly... Just wanted to say anyway you decide it's fine with me. Peace.

Reply:
Go with a simple "No URLs to display" for this PR.

If I feel strongly enough about having a different message when non-default filters are active, I can make another patch myself later :slightly_smiling_face:

Reply:
Let me just take a moment out of our busy days and appreciate the existence of actively maintained projects on GitHub. There are too many "other ones" around. I will make the changes as discussed once I'm awake!